App/src/main/java/User.java

```diff
@@ -0,0 +1,76 @@
+
+
+public class User {
+	/*For the User class the fields needed are:
+	 * Name, age, city, state, disabilities, and
+	 * number of families.
+     * 
+     * Class also contains multiple methods for creating and sending the
+     * information of the user
+     * 
+	 * */
+	private String Name;
```

Comment:
Check the indentation here and elsewhere in the file - these appear to not be aligned. (I know we've talked about this before and I don't remember what the outcome was. Ping me on chat when you have time and we can figure out what editor settings need to be changed, if any.